_src/static/sass/components/_headings.scss

```diff
@@ -0,0 +1,128 @@
+// ------------------------------
+// edX Pattern Library: Components - Headings
+
+// About: Contains base styling for headings, relying on Bourbon's mathematical scales for precision.
+
+// Scale:
+// 12, 14, 16, 18, 21, 24, 32, 43
+// ----------------------------
+
+// #GLOBALS
+// #HEADINGS
+// H1
+// H2
+// H3
+// H4
+// H5
+// H6
+// ----------------------------
+
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+.hd-1,
+.hd-2,
+.hd-3,
+.hd-4,
+.hd-5,
+.hd-6 {
+    font-family: $headings-base-font-family;
+    font-weight: $headings-base-font-weight;
+    color: $headings-base-color;
+
+    &.emphasized,
+    &.primary {
+        font-weight: $headings-primary-font-weight;
+        color: $headings-primary-color;
+    }
+
+    &.deemphasized,
+    &.secondary {
+        font-weight: $headings-secondary-font-weight;
+        color: $headings-secondary-color;
+    }
+}
+
+.hd-5,
+.hd-6 {
+    font-weight: $headings-primary-font-weight;
+    text-transform: uppercase;
+}
+
+
+// ----------------------------
+// #HEADINGS
+// ----------------------------
+// H1
```

Comment:
Also, as shown in the above example, please rename the headings to avoid any confusion between these classes and semantic heading tags (I want to avoid the "Do H2 styles only go on ``<h2>`` elements?" type of questions).